internal/pkg/cli/deploy.go

```diff
@@ -149,7 +151,8 @@ func newDeployOpts(vars deployVars) (*deployOpts, error) {
 				opts.newJobDeployer = func() (workloadDeployer, error) {
 					return newJobDeployer(opts)
 				}
-				o.deployWkld = opts
+				opts.name = workloadName
+				output = opts
```

Comment:
nit: i feel like this might be clearer if we get rid of the `output` decl. above and just `return opts` here

Reply:
hmm, good call.